app/windows/winget/winget_create.js

```diff
@@ -0,0 +1,144 @@
+/**
+ * Usage:
+ *
+ * 1. Ensure that Node.js and npm are installed on your system.
+ * 2. Run this script using the command `node winget_create.mjs <projectVersion> <directoryPath>`.
+ *
+ * Parameters:
+ * - projectVersion: The version of the project you want to fetch information for (e.g., 0.19.1).
+ * - directoryPath: The path to the directory where the generated YAML files should be stored (e.g., app/windows/winget/manifests).
+ *
+ * Ensure that you replace `<projectVersion>` with the actual project version, do not include the 'v'.
+ *
+ * Note:
+ * - The templates directory should contain the necessary YAML templates and should be located in the same directory as this script.
+ * - The manifests directory will be created if it does not exist, and it will contain the generated YAML files.
+ */
+
+const fetch = require('node-fetch');
+const fs = require('fs');
+const path = require('path');
+
+const args = process.argv.slice(2);
+
+if (args.length > 2) {
+  console.error('Usage: node winget_create.js <projectVersion>');
+  process.exit(1);
+}
+
+const projectVersion = args[0];
+
+const directoryPath = args[1];
+
+if (projectVersion.startsWith('v')) {
+  console.error('Please provide the project version without the "v" prefix.');
+  process.exit(1);
+}
+
+let sha256;
+
+async function createYAMLFiles(version, downloadURL, date, notesURL) {
+  const templates = [
+    'Headlamp.Headlamp.installer.yaml',
+    'Headlamp.Headlamp.locale.en-US.yaml',
+    'Headlamp.Headlamp.yaml',
+  ];
+
+  // use path to get the template directory
+  const templateDir = path.join(__dirname, 'templates/');
+
+  const manifestDir = path.join(directoryPath, '/', version, '/');
+
+  // create manifest version directory if it doesnt exist
+  if (!fs.existsSync(manifestDir)) {
+    fs.mkdirSync(manifestDir);
```

Comment:
Use the recursive option of this function, as it creates the missing directory paths for the user.
We should inform the user as well that we've created the directory, to inform of ongoing changes.

Reply:
added and also placed in a try catch to handle errors on creation